lib/cli.js

```diff
@@ -48,34 +49,58 @@ module.exports = function parseCLI(argv = process.argv.slice(2), help = HELP) {
 				"mode to avoid littering your file system with obsolete bundles");
 	}
 
+	let assetConfig = path.resolve(referenceDir, "faucet.config.js"); // TODO: customizable?
+	try {
+		assetConfig = require(assetConfig);
+	} catch(err) {
+		if(err.code !== "MODULE_NOT_FOUND") {
+			throw err;
+		}
+		assetConfig = {};
+	}
+
 	let { source, target, watch, fingerprint, sourcemaps, compact } = argv;
 	return {
 		referenceDir,
-		config: generateConfig(source, target, referenceDir),
+		config: generateConfig(source, target, referenceDir, assetConfig),
 		options: { watch, fingerprint, sourcemaps, compact }
 	};
 };
 
-function generateConfig(source, target, referenceDir) {
+function generateConfig(source, target, referenceDir, assetsConfig) {
 	let componentsDir = "./components"; // TODO: customizable
 	let baseURI = "/"; // TODO: customizable
 
-	return { // TODO: extensible (e.g. `js`)
-		watchDirs: [source, componentsDir], // TODO: extensible (e.g. `./lib`)
+	let { watchDirs, sass, plugins } = assetsConfig;
+
+	// merge `watchDirs`
```

Comment:
Why not `watchDirDefaults`, `sassDefaults` etc.?

Reply:
Is this still relevant, now that I've moved configuration into `config.js`?

Reply:
It's relevant, because comments like that scream for better variable names. Don't you think?

Reply:
Generally yes, that's an indicator for smell. In this case, however, I'd concluded something like `watchDirDefaults` and `sassDefaults` was unhelpful (you know I consider succinctness essential for pattern recognition) and that I'd much rather refactor the respective blocks in their entirety, as they seem awkwardly redundant and insufficienctly declarative.

Having said that, I'm not too invested here.